src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
-        valType: 'any',
-        dflt: {},
-        description: [
-            'Sets each group style.',
-            'For example, with `groups` set to *[\'a\', \'b\', \'a\', \'b\']*',
-            'and `style` set to *{ a: { marker: { color: \'red\' } }}',
-            'marker points in group *\'a\'* will be drawn in red.'
-        ].join(' ')
+        target: {
+            valType: 'any',
```

Comment:
Sounds good. I guess my discomfort is just in relying upon implicit conversion of keys to strings to sort out numerical identity. Like if your group were `0.2` and your style key were computed in JS as `0.3 - 0.1`, I don't think it would find the right style.